### DISCUSSION TITLE
After 100% he doesn't farming.

### DISCUSSION THREAD

**Discussion by davimussoi** [Chia-Network/chia-blockchain#5369] [Support]
I have a problem. After finishing the plot in 100% my batch goes to the External HD but it ends with .plot.tmp instead of just .tmp and not farm ... I tried to rename one of the batches only to .plot but the error message appeared in the GUI>
"Failed to open (invalid plots)
These plots are invalid, you might want to delete them. "
Does anyone know how this can be resolved, or how to prevent this from happening? Because it is very frustrating to waste 10 hours plotting a lot, and in the end it doesn't farm

**Comment by bill-95** [comment]
You have to wait for it to finish first (when it's finished, it will automatically start farming). You can't just rename it.

**Reply by davimussoi** [reply]
I found that the following error occurred in the system logs: Error The request could not be performed due to a device error.
At the time when I was copying the files to my external drive it was "disconnected" from windows. So I would like to know if there are any commands that I can do so that I try again to move the tmp files that are still on the SSD to the external HDD

**Reply by bill-95** [reply]
If the plotting process already stopped, I don't think there is a way to restart is currently.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
